_posts/2014-01-01-controller-web-api.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "Web API Applications"
+category: controller
+date: 2013-06-06 08:55:36
+order: 2
+---
+
+## Web API
+
+In the beta version of daemon mode, you can add your own http endpoints to amorphic's internal server object.
```

Comment:
this isn't unique to beta version correct? Can we remove the "beta version" mention?

Reply:
right - removed reference to beta version